Sources/JNI/JavaParameterConvertible.swift

```diff
@@ -23,6 +24,10 @@ extension Bool: JavaParameterConvertible {
         return JavaParameter(bool: (self) ? 1 : 0)
     }
 
+    public static func fromStaticField(of javaClass: JavaClass, id: jfieldID) throws -> Bool {
+        return jni.GetStaticBooleanField(of: javaClass, id: id) == 1
```

Comment:
return try jni.GetStaticBooleanField(of: javaClass, id: id) == JNI_TRUE

`GetStaticBooleanField` should throw